pkg/execute/executor.go

```diff
@@ -49,17 +35,26 @@ const (
 	lineLimitToShowFilter = 16
 )
 
+var (
+	clusterNameFlagRegex = regexp.MustCompile(`--cluster-name[=|\s]+\S+`)
```

Comment:
`\s             whitespace (== [\t\n\f\r ])`
`\S             not whitespace (== [^\t\n\f\r ])`

https://pkg.go.dev/regexp/syntax